next-sitemap.js

```diff
@@ -0,0 +1,26 @@
+/** @type {import('next-sitemap').IConfig} */
+
+function getHomepageUrl() {
+  const orgId = process.env.NEXT_PUBLIC_PLANNER_ORG_ID;
+  const org = require(`./orgs/${orgId}.json`);
+  return org.urls.homePageUrl.href;
+}
```

Comment:
I don't think that we should use the home page URL to define hostname of the planner web. We don't know that `reise.<homePageUrl>` is used by everyone. If it is needed I think we should add a new property, e.g. `hostname`, to the org file.

Reply:
Agreed, sounds like a more robust approach.